.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report a bug encountered while using Kubernetes release engineering tooling
+labels: kind/bug, sig/release, area/release-eng
```

Comment:
do these labels apply to bugs in cri-tools?

Reply:
maybe sig/node?

Reply:
+1 with sig/node and kind/bug

Reply:
Sounds good, adapted as suggested.